### ISSUE TITLE
Kucoin Cancell All Orders Ignoring Symbol Param

### ISSUE THREAD

**Issue by Kalindro** [ccxt/ccxt#8656]
- OS: Win 10
- Programming Language version: Python 3.7.9
- CCXT version: 1.43.46

Hi, news issue connected to my previous https://github.com/ccxt/ccxt/issues/8508 post, idk why I noticed it so late. I have an issue with Kucoin ` async def cancel_all_orders(self, symbol=None, params={})` where `Exchange.cancel_all_orders(symbol = "BTC/USDT")` cancels all existing orders even when I pass `symbol` parameter. I did some testing, ran the same code for Binance and Kucoin and for Binance it's working perfectly where for Kucoin it cancels my all 392 orders no matter which symbol is passed, seems ultra weird but I'm 99% sure it's not my fault.

**Comment by kroitor** [comment]
@Kalindro please, paste your verbose output:

````Python
print('CCXT Version:', ccxt.__version__)
exchange.verbose = True
await exchange.cancel_all_orders('BTC/USDT')
```

**Comment by Kalindro** [comment]
```
CCXT Version: 1.43.46

Request: GET https://openapi-v2.kucoin.com/api/v1/orders?status=active&symbol=SUN-USDT {'KC-API-KEY-VERSION': '2', 'KC-API-KEY': '60455ba0ebaa0100068a1f94', 'KC-API-TIMESTAMP': '1616052310119', 'KC-API-PASSPHRASE': 'H42whA6qC2hVeRC+z1GhDhuzQVSeGRk/PIfByugabJc=', 'KC-API-SIGN': 'jauxQdsZu2iiC14Z9DZzyByNxO4VkgcN9hDWE7Et2/Q=', 'User-Agent': 'python-requests/2.25.1', 'Accept-Encoding': 'gzip, deflate'} None

Response: GET https://openapi-v2.kucoin.com/api/v1/orders?status=active&symbol=SUN-USDT 200 {'Date': 'Thu, 18 Mar 2021 07:25:10 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=da90113dee9474ca13604a837eab3c1df1616052310; expires=Sat, 17-Apr-21 07:25:10 GMT; path=/; domain=.kucoin.com; HttpOnly; SameSite=Lax, AWSALB=Q5ttCaLZ0VI7BJf+AuLbBfbhr2sWAHyHdp2TkAL4LxneXHX+5c0NAwu/4h6R/yXkov1XT1hyoHvZw14FBsInJBIFDo6YxKmW4lKxJkfyJfIcOG7GIX4pyi6EW/bs; Expires=Thu, 25 Mar 2021 07:25:10 GMT; Path=/, AWSALBCORS=Q5ttCaLZ0VI7BJf+AuLbBfbhr2sWAHyHdp2TkAL4LxneXHX+5c0NAwu/4h6R/yXkov1XT1hyoHvZw14FBsInJBIFDo6YxKmW4lKxJkfyJfIcOG7GIX4pyi6EW/bs; Expires=Thu, 25 Mar 2021 07:25:10 GMT; Path=/; SameSite=None; Secure, __cfruid=495cc82a5e0d4f4b0e145b76e61dc3fd1cf4a041-1616052310; path=/; domain=.kucoin.com; HttpOnly; Secure; SameSite=None', 'X-Content-Type-Options': 'nosniff', 'X-XSS-Protection': '1; mode=block', 'Cache-Control': 'no-cache, no-store, max-age=0, must-revalidate', 'Expires': '0', 'X-Frame-Options': 'DENY', 'Strict-Transport-Security': 'max-age=9800; includeSubDomains; preload', 'CF-Cache-Status': 'DYNAMIC', 'cf-request-id': '08e5d2686c000021993fa46000000001', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '631cb9ba4c832199-DUS', 'Content-Encoding': 'gzip'} {"code":"200000","data":{"currentPage":1,"pageSize":50,"totalNum":5,"totalPage":1,"items":[{"id":"6052fff1d2b21e00060ec7ae","symbol":"SUN-USDT","opType":"DEAL","type":"limit","side":"buy","price":"20.48281","size":"1.196","funds":"0","dealFunds":"0","dealSize":"0","fee":"0","feeCurrency":"USDT","stp":"","stop":"","stopTriggered":false,"stopPrice":"0","timeInForce":"GTC","postOnly":false,"hidden":false,"iceberg":false,"visibleSize":"0","cancelAfter":0,"channel":"API","clientOid":"c81a7127-e523-4f59-913d-9144037a69f8","remark":null,"tags":null,"isActive":true,"cancelExist":false,"createdAt":1616052209000,"tradeType":"TRADE"},{"id":"6052fff02622c30006b9300c","symbol":"SUN-USDT","opType":"DEAL","type":"limit","side":"buy","price":"20.585739","size":"1.196","funds":"0","dealFunds":"0","dealSize":"0","fee":"0","feeCurrency":"USDT","stp":"","stop":"","stopTriggered":false,"stopPrice":"0","timeInForce":"GTC","postOnly":false,"hidden":false,"iceberg":false,"visibleSize":"0","cancelAfter":0,"channel":"API","clientOid":"a4169060-7781-4f05-a877-0b99c631bf2e","remark":null,"tags":null,"isActive":true,"cancelExist":false,"createdAt":1616052208000,"tradeType":"TRADE"},{"id":"6052fff05057fd000658487f","symbol":"SUN-USDT","opType":"DEAL","type":"limit","side":"buy","price":"20.534274","size":"1.196","funds":"0","dealFunds":"0","dealSize":"0","fee":"0","feeCurrency":"USDT","stp":"","stop":"","stopTriggered":false,"stopPrice":"0","timeInForce":"GTC","postOnly":false,"hidden":false,"iceberg":false,"visibleSize":"0","cancelAfter":0,"channel":"API","clientOid":"e5365eca-f661-4927-8500-315efa011d36","remark":null,"tags":null,"isActive":true,"cancelExist":false,"createdAt":1616052208000,"tradeType":"TRADE"},{"id":"6052ffef7eda2300068534c2","symbol":"SUN-USDT","opType":"DEAL","type":"limit","side":"buy","price":"20.688667","size":"1.196","funds":"0","dealFunds":"0","dealSize":"0","fee":"0","feeCurrency":"USDT","stp":"","stop":"","stopTriggered":false,"stopPrice":"0","timeInForce":"GTC","postOnly":false,"hidden":false,"iceberg":false,"visibleSize":"0","cancelAfter":0,"channel":"API","clientOid":"b6fcba14-6c64-4b9d-be93-3157a4a24faa","remark":null,"tags":null,"isActive":true,"cancelExist":false,"createdAt":1616052207000,"tradeType":"TRADE"},{"id":"6052ffefca9e3c00065c7949","symbol":"SUN-USDT","opType":"DEAL","type":"limit","side":"buy","price":"20.637203","size":"1.196","funds":"0","dealFunds":"0","dealSize":"0","fee":"0","feeCurrency":"USDT","stp":"","stop":"","stopTriggered":false,"stopPrice":"0","timeInForce":"GTC","postOnly":false,"hidden":false,"iceberg":false,"visibleSize":"0","cancelAfter":0,"channel":"API","clientOid":"6b268ab8-0b68-437b-9e23-d50d5816c06e","remark":null,"tags":null,"isActive":true,"cancelExist":false,"createdAt":1616052207000,"tradeType":"TRADE"}]}}

Request: DELETE https://openapi-v2.kucoin.com/api/v1/orders {'KC-API-KEY-VERSION': '2', 'KC-API-KEY': '60455ba0ebaa0100068a1f94', 'KC-API-TIMESTAMP': '1616052310617', 'Content-Type': 'application/json', 'KC-API-PASSPHRASE': 'H42whA6qC2hVeRC+z1GhDhuzQVSeGRk/PIfByugabJc=', 'KC-API-SIGN': 'cS6satrGXOuAbNsil7tz71gNvF4Qklkg9RAUVldXp8E=', 'User-Agent': 'python-requests/2.25.1', 'Accept-Encoding': 'gzip, deflate'} {"symbol":"BTC-USDT"}

Response: DELETE https://openapi-v2.kucoin.com/api/v1/orders 200 {'Date': 'Thu, 18 Mar 2021 07:25:10 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=da90113dee9474ca13604a837eab3c1df1616052310; expires=Sat, 17-Apr-21 07:25:10 GMT; path=/; domain=.kucoin.com; HttpOnly; SameSite=Lax, AWSALB=WvH4WPrISTphGsT9JhqlDQ+BYBSox4ryxIt2w0URypCWd15CjrJglhNtESr2kgzMkP1QXo1WkYQE+NuxCJvZDQCyhomft5ricRVWVymRw/oGzRr0T4TtEnZceZF+; Expires=Thu, 25 Mar 2021 07:25:10 GMT; Path=/, AWSALBCORS=WvH4WPrISTphGsT9JhqlDQ+BYBSox4ryxIt2w0URypCWd15CjrJglhNtESr2kgzMkP1QXo1WkYQE+NuxCJvZDQCyhomft5ricRVWVymRw/oGzRr0T4TtEnZceZF+; Expires=Thu, 25 Mar 2021 07:25:10 GMT; Path=/; SameSite=None; Secure, __cfruid=495cc82a5e0d4f4b0e145b76e61dc3fd1cf4a041-1616052310; path=/; domain=.kucoin.com; HttpOnly; Secure; SameSite=None', 'Strict-Transport-Security': 'max-age=9800; includeSubDomains; preload', 'CF-Cache-Status': 'DYNAMIC', 'cf-request-id': '08e5d26a5d000021992131f000000001', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '631cb9bd6bdc2199-DUS', 'Content-Encoding': 'gzip'} {"code":"200000","data":{"cancelledOrderIds":["6052ffef7eda2300068534c2","6052ffefca9e3c00065c7949","6052fff02622c30006b9300c","6052fff05057fd000658487f","6052fff1d2b21e00060ec7ae","6052fff663bd780006960b82","6052fff712ec1700066a7ca3","6052fff712ec1700066a7d8f","6052fff79f9c050006148201","6052fff81220fc0006f01298","6052fffb1220fc0006f023bb","6052fffb1220fc0006f025a9","6052fffc1220fc0006f0291c","6052fffc5057fd0006587c33","6052fffcd2b21e00060ef906","60530000d2b21e00060f0c0e","6053000112ec1700066aac40","60530001ca9e3c00065cc9b8","60530001d2b21e00060f0f7b","605300029f9c05000614b53a","60530005d2b21e00060f1e0a","605300067eda23000685994e","60530006ca9e3c00065cdddc","60530006d2b21e00060f2073","605300077eda230006859cd3","6053000a43af58000643a4c0","6053000ad2b21e00060f30c6","6053000b12ec1700066ad574","6053000b43af58000643a7dc","6053000bb5ab39000634cfd6","6053000f5057fd000658cbfb","6053000f63bd780006967662","6053000fb5ab39000634e3eb","605300101220fc0006f079fe","6053001063bd780006967bc9","6053001463bd780006968e68","6053001512ec1700066b0056","60530015d2b21e00060f5f82","60530015f6e8e80006057dc9","60530016ca9e3c00065d1f11","6053001a7eda23000685ebb2","6053001a9f9c050006151902","6053001ad2b21e00060f7510","6053001b2622c30006b9eaed","6053001bb5ab3900063512f9","6053001edfaee70006fbd5d5","6053001f43af58000643fdfa","6053001f63bd78000696b9f7","605300202622c30006b9ffe4","60530020b5ab390006352821","605300245057fd0006592026","605300249f9c050006153e9a","60530024ca9e3c00065d58a1","6053002563bd78000696cf69","6053002563bd78000696d0cd","605300289f9c0500061550a9","605300292622c30006ba2437","605300295057fd0006593341","6053002963bd78000696de85","6053002af6e8e8000605d041","6053002d5057fd0006594be7","6053002df6e8e8000605e2b6","6053002e1220fc0006f0fdbd","6053002e7eda230006864149","6053002ec7fa9f00061385f6","6053002fb5ab3900063569d4","6053002fb5ab390006356b1b","6053002fdfaee70006fc1cd6","605300331220fc0006f1103b","60530034b5ab390006357dd6","605300355057fd0006596b77","60530035c7fa9f000613a183","605300367eda230006866076","605300399f9c0500061599a2","60530039dfaee70006fc4587","6053003ad2b21e00060ff67d","6053003ad2b21e00060ff780","6053003bd2b21e00060ffe2b","6053003e9f9c05000615af93","6053003f63bd780006973ca4","6053003fca9e3c00065dc53a","6053003fca9e3c00065dc65e","60530040dfaee70006fc5e18","605300507eda23000686c5de","6053005112ec1700066bf39c","605300515057fd000659d8be","60530051b5ab39000635eb82","6053005243af58000644ce9a"]}}
```

**Comment by Kalindro** [comment]
@kroitor 

```
Request: DELETE https://openapi-v2.kucoin.com/api/v1/orders {'KC-API-KEY-VERSION': '2', 'KC-API-KEY': '60455ba0ebaa0100068a1f94', 'KC-API-TIMESTAMP': '1616052310617', 'Content-Type': 'application/json', 'KC-API-PASSPHRASE': 'H42whA6qC2hVeRC+z1GhDhuzQVSeGRk/PIfByugabJc=', 'KC-API-SIGN': 'cS6satrGXOuAbNsil7tz71gNvF4Qklkg9RAUVldXp8E=', 'User-Agent': 'python-requests/2.25.1', 'Accept-Encoding': 'gzip, deflate'} {"symbol":"BTC-USDT"}

Response: DELETE https://openapi-v2.kucoin.com/api/v1/orders 200 {'Date': 'Thu, 18 Mar 2021 07:25:10 GMT', 'Content-Type': 'application/json', 'Transfer-Encoding': 'chunked', 'Connection': 'keep-alive', 'Set-Cookie': '__cfduid=da90113dee9474ca13604a837eab3c1df1616052310; expires=Sat, 17-Apr-21 07:25:10 GMT; path=/; domain=.kucoin.com; HttpOnly; SameSite=Lax, AWSALB=WvH4WPrISTphGsT9JhqlDQ+BYBSox4ryxIt2w0URypCWd15CjrJglhNtESr2kgzMkP1QXo1WkYQE+NuxCJvZDQCyhomft5ricRVWVymRw/oGzRr0T4TtEnZceZF+; Expires=Thu, 25 Mar 2021 07:25:10 GMT; Path=/, AWSALBCORS=WvH4WPrISTphGsT9JhqlDQ+BYBSox4ryxIt2w0URypCWd15CjrJglhNtESr2kgzMkP1QXo1WkYQE+NuxCJvZDQCyhomft5ricRVWVymRw/oGzRr0T4TtEnZceZF+; Expires=Thu, 25 Mar 2021 07:25:10 GMT; Path=/; SameSite=None; Secure, __cfruid=495cc82a5e0d4f4b0e145b76e61dc3fd1cf4a041-1616052310; path=/; domain=.kucoin.com; HttpOnly; Secure; SameSite=None', 'Strict-Transport-Security': 'max-age=9800; includeSubDomains; preload', 'CF-Cache-Status': 'DYNAMIC', 'cf-request-id': '08e5d26a5d000021992131f000000001', 'Expect-CT': 'max-age=604800, report-uri="https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct"', 'Server': 'cloudflare', 'CF-RAY': '631cb9bd6bdc2199-DUS', 'Content-Encoding': 'gzip'} {"code":"200000","data":{"cancelledOrderIds":["6052ffef7eda2300068534c2","6052ffefca9e3c00065c7949","6052fff02622c30006b9300c","6052fff05057fd000658487f","6052fff1d2b21e00060ec7ae","6052fff663bd780006960b82","6052fff712ec1700066a7ca3","6052fff712ec1700066a7d8f","6052fff79f9c050006148201","6052fff81220fc0006f01298","6052fffb1220fc0006f023bb","6052fffb1220fc0006f025a9","6052fffc1220fc0006f0291c","6052fffc5057fd0006587c33","6052fffcd2b21e00060ef906","60530000d2b21e00060f0c0e","6053000112ec1700066aac40","60530001ca9e3c00065cc9b8","60530001d2b21e00060f0f7b","605300029f9c05000614b53a","60530005d2b21e00060f1e0a","605300067eda23000685994e","60530006ca9e3c00065cdddc","60530006d2b21e00060f2073","605300077eda230006859cd3","6053000a43af58000643a4c0","6053000ad2b21e00060f30c6","6053000b12ec1700066ad574","6053000b43af58000643a7dc","6053000bb5ab39000634cfd6","6053000f5057fd000658cbfb","6053000f63bd780006967662","6053000fb5ab39000634e3eb","605300101220fc0006f079fe","6053001063bd780006967bc9","6053001463bd780006968e68","6053001512ec1700066b0056","60530015d2b21e00060f5f82","60530015f6e8e80006057dc9","60530016ca9e3c00065d1f11","6053001a7eda23000685ebb2","6053001a9f9c050006151902","6053001ad2b21e00060f7510","6053001b2622c30006b9eaed","6053001bb5ab3900063512f9","6053001edfaee70006fbd5d5","6053001f43af58000643fdfa","6053001f63bd78000696b9f7","605300202622c30006b9ffe4","60530020b5ab390006352821","605300245057fd0006592026","605300249f9c050006153e9a","60530024ca9e3c00065d58a1","6053002563bd78000696cf69","6053002563bd78000696d0cd","605300289f9c0500061550a9","605300292622c30006ba2437","605300295057fd0006593341","6053002963bd78000696de85","6053002af6e8e8000605d041","6053002d5057fd0006594be7","6053002df6e8e8000605e2b6","6053002e1220fc0006f0fdbd","6053002e7eda230006864149","6053002ec7fa9f00061385f6","6053002fb5ab3900063569d4","6053002fb5ab390006356b1b","6053002fdfaee70006fc1cd6","605300331220fc0006f1103b","60530034b5ab390006357dd6","605300355057fd0006596b77","60530035c7fa9f000613a183","605300367eda230006866076","605300399f9c0500061599a2","60530039dfaee70006fc4587","6053003ad2b21e00060ff67d","6053003ad2b21e00060ff780","6053003bd2b21e00060ffe2b","6053003e9f9c05000615af93","6053003f63bd780006973ca4","6053003fca9e3c00065dc53a","6053003fca9e3c00065dc65e","60530040dfaee70006fc5e18","605300507eda23000686c5de","6053005112ec1700066bf39c","605300515057fd000659d8be","60530051b5ab39000635eb82","6053005243af58000644ce9a"]}}
```

**Comment by kroitor** [comment]
@Kalindro thank you! I've pushed a fix, please upgrade to version 1.43.69 (the new build will arrive in ~20 minutes) and let us know if you still have this issue. Thx again for your feedback!

**Comment by Kalindro** [comment]
@kroitor Thanks, will post if issue persists!
